sdk/evaluation/azure-ai-evaluation/tests/unittests/test_lazy_imports.py

```diff
@@ -0,0 +1,127 @@
+"""Tests for lazy import behavior of optional dependencies."""
+
+import sys
+import unittest
+from io import StringIO
+import importlib
+
+
+class TestLazyImports(unittest.TestCase):
+    """Test lazy import behavior for optional dependencies."""
+
+    def test_no_messages_during_module_import(self):
```

Comment:
Can you run this test only when semantic_kernel is not installed ? Like it is done here https://github.com/ahibrahimm/azure-sdk-for-python/blob/9da450d6739b7cb67fb24f1576b2b93ee55cc7b4/sdk/evaluation/azure-ai-evaluation/tests/converters/ai_agent_converter/test_sk_agent_converter_internals.py